lib/ansible/modules/database/misc/elasticsearch_plugin.py

```diff
@@ -43,6 +43,11 @@
             - "Timeout setting: 30s, 1m, 1h..."
             - Only valid for Elasticsearch < 5.0. This option is ignored for Elasticsearch > 5.0.
         default: 1m
+    force:
+        description:
+            - "Give the possibility to force a plugin installation even with addition permission needed"
```

Comment:
I would change the wording on this to align more with the [Elatic docs](https://www.elastic.co/guide/en/elasticsearch/plugins/6.3/_other_command_line_parameters.html#_batch_mode) and indicate what this parameter is doing under the hood. Something along the lines of "Force batch mode when installing plugins. This is only necessary if a plugin requires additional permissions and console detection fails."

Reply:
Good idea. I've changed this.